docs/FAQ.md

```diff
@@ -0,0 +1,58 @@
+# Frequently asked questions
+
+### Why does my branch not move to the new commit after `jj new/commit`?
+
+Jujutsu's branches are akin to Git Tags, they always point to a single commit.
```

Comment:
I think this is true of git's branches as well. Also, I wouldn't mention git tags, they'd deserve their own FAQ question since jj treats them differently from branches (when a commit is rebased, branches move with it but tags don't).

Reply:
The point I want to get across is, that branches don't move with new commits. I think that a Git Tag, conveys the pointer like behavior for Git users.

Reply:
Referring to Git doesn't help those of us that don't know either Git or JJ.

Reply:
I also wouldn't say that they're like git tags because git tags don't move when you rewrite a commit.

Regarding users who are not familiar with git: I agree with @joyously that we shouldn't assume that users are familiar with git, but I would also think that such users wouldn't even read this entry because they would have no reason to expect that branches automatically move forward. Maybe we could say something like this:

> If you're familiar with Git, you might expect the current branch to move forward when you commit. However, jj doesn't have a concept of a "current branch". ...